test/runner.py

```diff
@@ -27,15 +27,6 @@ def prepare_docker() -> docker.DockerClient:
     client = docker.DockerClient(base_url="unix://var/run/docker.sock")
 
     # Network creation
-    ipv4_net = client.networks.create(
```

Comment:
Why it's unnecessary?

Reply:
there's `net6` just few lines below, so this wasn't necessary

Reply:
But it doesn't contain an IPv4 range. How is it possible that it still works?

Reply:
Basically because net6 contains both :)